app/models/donation.rb

```diff
@@ -9,5 +9,7 @@ class Donation < ActiveRecord::Base
 
   scope :search, ->(keyword) { joins(:donor).where("name ILIKE ?", "%#{keyword}%") }
 
-  scope :donations_per_year, ->(year) { where("EXTRACT(YEAR FROM created_at) = ?", year) }
+  scope :donations_per_year, ->(year) { where("EXTRACT(YEAR FROM donations.created_at) = ?", year) }
```

Comment:
I would name this `.donations_for_year`, as "per year" might imply that we're getting results for several years.